pitch.md

```diff
@@ -23,38 +37,125 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+Like LaTeX the programmer would write text to a file. 
+The programmer types out what would be on each line of the tableau tree,
+using special characters for logic symbols.
+The programmer should also have a way to draw a check mark at the end of each line of the tree.
+Tableau trees are binary trees so the programmer can also specify if the tree branches off.
+
+One possible syntax is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+to draw
+
+![tree](/images/tree.png)
+
+which is similar to how qtree, a LaTeX package for trees, structures code but this doesn't require entering math mode using \$.
+Anthony mentioned in his critique that the backslash may not be necessary in this language.
```

Comment:
Oh, I now understand the reason you use slashes. What if you enclose variable names in quotation marks? Not sure if this makes parsing more difficult or easier for you :)